geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/internal/SqlHandler.java

```diff
@@ -63,6 +76,57 @@ public SqlHandler(InternalCache cache, String regionName,
         dataSourceName -> JNDIInvoker.getDataSource(dataSourceName));
   }
 
+  protected void compareTableMetadataWithMapping(TableMetaDataManager manager,
+      RegionMapping mapping, String regionName) throws Exception {
+    Set<String> jdbcNames = new HashSet<>();
+    Set<String> jdbcTypes = new HashSet<>();
+
+    System.out.println("Field names in mapping:");
+    for (FieldMapping fieldMapping : mapping.getFieldMappings()) {
+      jdbcNames.add(fieldMapping.getJdbcName());
+      jdbcTypes.add(fieldMapping.getJdbcType());
+      System.out.println(fieldMapping.getJdbcName() + " - " + fieldMapping.getJdbcType());
+    }
+
+    System.out.println("\n");
```

Comment:
get rid of all the System.out.printlns in this method

Reply:
Same as above.